app/app.js

```diff
@@ -0,0 +1,61 @@
+import $ from "jquery";
+import * as tournament from "./tournament/tournament";
+import * as player from "./tournament/player";
+
+var mTournament = new tournament.Tournament();
+
+$(document).ready(function() {
+    $("#buttonAdd").click(addPlayerIfValid);
+    $("#buttonCreate").click(generateTournament);
+});
+
+function addPlayerIfValid() {
```

Comment:
This is more than I would expect in app.js, it should just initialize things and kick off the app.